lib/models/offerings_wrapper.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+import 'offering_wrapper.dart';
+
+part 'offerings_wrapper.freezed.dart';
+part 'offerings_wrapper.g.dart';
+
+@freezed
+
+/// This class contains all the offerings configured in RevenueCat dashboard.
+/// For more info see https://docs.revenuecat.com/docs/entitlements
+class Offerings with _$Offerings {
```

Comment:
I think there might be an issue with the mapping for this class, when testing using [the MagicWeather app](https://github.com/RevenueCat/purchases-flutter/tree/main/revenuecat_examples/MagicWeather) I'm running into `type '_InternalLinkedHashMap<Object?, Object?>' is not a subtype of type 'Map<String, dynamic>'` when trying to `getOfferings`. Does it work correctly for you? Maybe I'm missing a step

Reply:
Great catch, below are the fixes